src/docker/DockerCompose.js

```diff
@@ -98,14 +101,22 @@ class DockerCompose {
    * @return {Promise<void>}
    */
   async up(envs) {
-    await this.throwErrorIfNotInstalled();
-
-    try {
-      await dockerCompose.upAll({
-        ...this.getOptions(envs),
-      });
-    } catch (e) {
-      throw new DockerComposeError(e);
+    const composeV2Installed = await this.isComposeV2Installed();
```

Comment:
We need an option to specify which version of the compose to use. Otherwise, it will be confusion due to errors during the build and start services.